src/components/VideoPlayer/VideoPlayerControls/VolumeButton/index.js

```diff
@@ -42,15 +42,20 @@ function VolumeButton({style, small}) {
         setSliderHeight(e.nativeEvent.layout.height);
     };
 
+    const gestureEventHandler = (event) => {
```

Comment:
This name is against the Expensify guidelines. I believe it should be `handleGestureEvent`.

Reply:
Or, even better, `changeVolumeOnPan`

Reply:
@kowczarz Bump! 🙂